ui/src/forms/FormProperty.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <label v-if="property && !isHidden" class="form-property">
+  <label class="form-property">
```

Comment:
Not necessary, since Shaperone now supports the `dash:hidden` as well as explicit hide/show feature, used fox alternating `xone` visibility